src/Illuminate/Auth/Access/Gate.php

```diff
@@ -39,35 +39,35 @@ class Gate implements GateContract
     /**
      * All of the defined abilities.
      *
-     * @var array
+     * @var array<string, callable>
      */
     protected $abilities = [];
 
     /**
      * All of the defined policies.
      *
-     * @var array
+     * @var array<class-string<\Illuminate\Database\Eloquent\Model>, class-string>
      */
     protected $policies = [];
 
     /**
      * All of the registered before callbacks.
      *
-     * @var array
+     * @var callable[]
```

Comment:
Quite similar to the `$afterCallbacks`:
```suggestion
     * @var (callable(\Illuminate\Contracts\Auth\Authenticatable, string, array): bool|null)[]
```

Reply:
Applied